contributing_to_docs/doc_guidelines.adoc

```diff
@@ -762,9 +761,7 @@ Now using project "my-project" on server "https://openshift.example.com:6443".
 ----
 ....
 
-* To mark up command syntax, use the code block and wrap any replaceable values in
-angle brackets (`<>`) with the required command parameter, using underscores
-(`_`) between words as necessary for legibility. For example:
+* To mark up command syntax, use the code block and wrap any replaceable values in angle brackets (`<>`) with the required command parameter, using underscores (`_`) between words as necessary for legibility. For example:
```

Comment:
Removed hard wrap. Trying to decide if it would be best to say something about paths here explicitly, or if the statement in this bullet paired with the [example in the quick reference table](https://github.com/openshift/openshift-docs/pull/32197/files#diff-719cf3b36e5cfdc45393ebde8ff09fd9a56c239b5671fe1cea5b7f364d647125R1331) is sufficient.

Reply:
I think it would be good to mention the desired style, and explicitly to avoid `/path/to/` style.

Reply:
Do you think as part of this bullet, or as a separate item?

Reply:
FWIW, the Supplementary Guide just leaves it as simply `<path>`.

Reply:
> Do you think as part of this bullet, or as a separate item?

@jeana-redhat New bullet seems good.

Reply:
Ok, will work on that. I suppose the `<path>` approach makes sense in a lot of cases, as long as the instructions are clear. However, I have run into examples while looking at this that have two paths in a command - so maybe being more verbose is safer? Example:
```
oc create secret tls <certificate> --cert=</path/to/cert.crt> --key=</path/to/cert.key> -n openshift-config
```